### ISSUE TITLE
Add arrow navigation to the editor toolbars

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#631]
As mentioned on #628 the editor toolbars should use arrow navigation. It should work in the same way it's going to be implemented for the inserter menu, see #515.

Tab, Down, and Right: to navigate to the next control
Shift+Tab, Up, and Left: to navigate to the previous control

**Edit: actually, I'm not sure about Tab. If implemented as ARIA toolbars, there should be just arrows navigation.**

Also, both tabbing and arrowing should be constrained within the toolbar.

Worth noting this it's a standard feature of the current Editor: once you focus the toolbar with Alt+F10 all the buttons can be navigated with both the Tab key and the arrow keys.

**Comment by afercia** [comment]
There is now just one, fixed, toolbar at the top but this issue still applies.

**Comment by afercia** [comment]
Arrow navigation is now implemented. Further improvements discussed on https://github.com/WordPress/gutenberg/issues/3383
